Detectors/TRD/reconstruction/src/CruRawReader.cxx

```diff
@@ -244,6 +244,7 @@ bool CruRawReader::parseDigitHCHeaders(int hcid)
   // mHBFoffset32 is the current offset into the current buffer,
   //
   mDigitHCHeader.word = mHBFPayload[mHBFoffset32++];
+  mPreTriggerPhase = 0; // set to zero incase there is no half header with the pretriggerphase, which is optional
```

Comment:
would it not make sense to set it to some "invalid" value so that in QC one can decide if/how to use the digit?

Reply:
I agree, but 0 keeps the same situation as old digits.
I have changed it to 4 which is invalid, for reasons listed in the code.